SalesforceCommerceCloud/pwa-kit
# Replace `bundlesize` with `bundlesize2`

## Problem

The `bundlesize` module through transitive dependencies requires prebuilt binaries to be installed (for a library called `iltorb`). Unfortunately there are no prebuilt binaries available for `node 14` which is now the default version of node we use. The result is that if you don't have all the correct build tools installed, then the build will file when trying to build these binaries locally. 

To solve this problem, the maintainer of `bundlesize` fixed this issue (my making brotli-size optional, as there is internal support in node since v10), but only made it available in a beta channel, which was later moved to to live under the `bundlesize2` name. I believe this was done because the maintainer didn't want to move to a `1.x` version. 


 **GUS**: [W-9755417](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07AH000000U4klYAC/view)
 **Linked PRs**: None

## How to test-drive this PR
- Check that the root module that is causing the issue is gone. Run the below command and ensure that it returns `(empty)`.
```
npm ls iltorp
```

## Changes
- Switch out `bundlesize` for `bundlesize2`

## TODO
- [ ] Test on windows vm
